src/xdocs/contributing.xml

```diff
@@ -164,6 +164,9 @@
         Small changes of configuration files, documentation fixes, etc. can be contributed by
         starting the commit message with one of "minor:", "config:", "infra:", "doc:",
         "dependency:" or "spelling:", followed by a space and a brief single-line message.
+        Changes that improves performance or is connected to another PR the commit message
+        should be "supplemental: .... for Issue #XXXX" where XXXX is issue number is to which
+        the PRs aim to solve.
```

Comment:
I think that using the existing PR where this issue was raised is a better of example of when we need to use this prefix, instead of `performance`.  Please take a look at #8897; it was to fix violations in the project that were caused by the creation of a new check, and was merged before the main PR.  The advantage of using this prefix is that we can quickly merge changes that are required by another PR (and have greater granularity in commits), and `supplemental` gives us a way to do that, while also allowing us to ignore the associated changes in release notes.

I would write something here to highlight this. PRs/ commits using this prefix will be used to support other, more significant changes.

Reply:
Done.

Reply:
perhaps `... where "XXXX" refers to the issue that this commit is supplementing.`? Or maybe  `... where "XXXX" refers to the issue that this commit is supporting.`? Just a thought. Or maybe even `... where "XXXX" refers to the issue that requires this supplemental commit.`

Reply:
Yup, these are definitely better and would remove any confusion. Thanks Sir. Done.

Reply:
Resolved.